### ISSUE TITLE
Add a link to error output

### ISSUE THREAD

**Issue by octref** [vuejs/eslint-plugin-vue#439]
See https://github.com/vuejs/vetur/issues/738 and https://github.com/vuejs/vetur/issues/261

Many new Vue users are having a hard time figuring out the meaning of the error message for `v-bind:key`.

I think it would be nice if eslint-plugin-vue offers a url for each rule, this way I can render the URL and people can find more information about each error there.

![eslint](https://user-images.githubusercontent.com/4033249/37744097-a0f38fb8-2d2a-11e8-8e8a-2ae5c49c9109.gif)

**Comment by octref** [comment]
Also, I suggest adding links to either official doc or style guide in each rule's markdown file. For example, for this one: https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/valid-v-bind.md

I think both https://vuejs.org/v2/guide/list.html#key and https://vuejs.org/v2/style-guide/#Keyed-v-for-essential should be linked in a section like `Related Links`.

**Comment by michalsnik** [comment]
Good idea @octref with putting links in documentation :)

But when it comes to displaying rules' URLs in vuetur's tooltips, you can already take advantage of `eslint-rule-documentation`. Check it out [here](https://npm.runkit.com/eslint-rule-documentation). Usage is as simple as:

```js
const getRuleURI = require('eslint-rule-documentation');

getRuleURI('vue/require-v-for-key');
```

**Comment by chrisvfritz** [comment]
Cool idea! Any chance of the links being openable, like via a shortcut?

**Comment by octref** [comment]
> eslint-rule-documentation

Very nice. Thanks for the tip.

> Cool idea! Any chance of the links being openable, like via a shortcut?

![image](https://user-images.githubusercontent.com/4033249/37784946-2ba71680-2db6-11e8-8eea-3344cd77e383.png)

This is using `Hover` since it allows markdown (and therefore linkified links).
When https://github.com/Microsoft/language-server-protocol/issues/250 is addressed I can use the markdown for the error message too.

**Comment by michalsnik** [comment]
Cool :) So if the rest is on your side now, let me set the definition of done for us, regarding your suggestion:
- [ ] Update rules documentations with links to style guide

**Comment by octref** [comment]
Will upgrade the deps and get it soon. https://github.com/vuejs/vetur/issues/849
Thanks for your work!

**Comment by octref** [comment]
Unfortunately this still depends on upstream changes in: https://github.com/Microsoft/vscode/issues/54272

Otherwise, I cannot render the link in diagnostic hover (the one you saw was on normal hover).

**Comment by sahidmunoz892-oss** [comment]
> See [vuejs/vetur#738](https://github.com/vuejs/vetur/issues/738) and [vuejs/vetur#261](https://github.com/vuejs/vetur/issues/261)
> 
> Many new Vue users are having a hard time figuring out the meaning of the error message for `v-bind:key`.
> 
> I think it would be nice if eslint-plugin-vue offers a url for each rule, this way I can render the URL and people can find more information about each error there.
> 
> ![eslint](https://user-images.githubusercontent.com/4033249/37744097-a0f38fb8-2d2a-11e8-8e8a-2ae5c49c9109.gif) [ ![eslint](https://user-images.githubusercontent.com/4033249/37744097-a0f38fb8-2d2a-11e8-8e8a-2ae5c49c9109.gif) ](https://user-images.githubusercontent.com/4033249/37744097-a0f38fb8-2d2a-11e8-8e8a-2ae5c49c9109.gif) [ ](https://user-images.githubusercontent.com/4033249/37744097-a0f38fb8-2d2a-11e8-8e8a-2ae5c49c9109.gif)
